l10n_ro_hr/hr_contract_view.xml

```diff
@@ -62,30 +62,26 @@
 				</field>
```

Comment:
In Contract poti pune si invisible pentru date_end in functie de period_type...daca e pe perioada nedeterminata, adica bifat period_type, nu ar trebui sa afiseze date_end...